contracts/SpokePool.sol

```diff
@@ -795,19 +795,19 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
         bytes32 relayHash,
         uint256 fillAmount,
         uint256 repaymentChainId,
-        uint64 relayerFeePct,
+        uint64 appliedRelayerFeePct,
         RelayData memory relayData,
         bool isSlowRelay
     ) internal {
         emit FilledRelay(
-            relayHash,
```

Comment:
I removed this to get around a `stack too deep error` but also realized we don't use `relayHash` in the current client code so its a reasonable delete